File: DotNetGrc/Grc/Ast/Node/Type/TypeReturnNothingT.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Grc.Ast.Visitor;

namespace Grc.Ast.Node.Type
{
	public class TypeReturnNothingT : TypeReturnBase
	{
		public TypeReturnNothingT(string keyNothing, int line, int pos)
			: base(keyNothing, line, pos)
		{
		}

		public override void Accept(IVisitor v)
		{
			v.Visit(this);
		}
	}
}

File: Parser/src/main/java/k31/grc/ast/node/type/TypeReturnNothingT.java
package k31.grc.ast.node.type;

import k31.grc.ast.visitor.Visitor;

public class TypeReturnNothingT extends TypeReturnBase {

	public TypeReturnNothingT(String text) {
		super(text);
	}

	@Override
	public void accept(Visitor v) {

		v.visit(this);
	}
}
